src/views/appShell/AppLogIn/components/AuthorithiesChecker.tsx

```diff
@@ -0,0 +1,46 @@
+export function checkAuthority(privilege: SECURITY_PRIVILEGES, securityGroupName: string) {
+    if (securityGroupName == null || privilege == null) {
+        return false;
+    }
+    return sessionStorage.getItem('authorities').includes(`${privilege}@${securityGroupName}`);
```

Comment:
interpret authorities before checking includes

Reply:
I think we can interpret authorities like this : JSON.parse(sessionStorage.getItem('authorities')).includes(`${privilege}@${securityGroupName}`);

so like this we can parse all the informations from JWToken and to an object and then check if the object include the parameters passed ;